docs/design_proposal.md

```diff
@@ -69,4 +69,4 @@ Even if we don't provide an SDK layer for a language, we still need to provide a
 
 ## 4. CI/CD convention
 Each sample application and layer source folder must have a one-click script `run.sh`, to build and deploy OpenTelemetry Lambda application in a uniform manner. It is not only providing a good user experience but also for building a generic CI/CD workflow, github action can simply run tests and publish Lambda layer no matter what the stack tool(Terraform, CloudFormation), or language(Java, Python) the sample application is. 
-See an example for publishing [Python layer](../python/src/).
\ No newline at end of file
+See an example for publishing [Python layer](../python/src/).
```

Comment:
What is the change on this line? I can't tell from the GitHub diff 😓

Reply:
I didn't intend to make a change here. I'm uncertain what it is picking up

Reply:
Hm you probably want to do something like `git checkout origin/main -- docs/design_proposal.md` to rest the changes, and then apply your fix above again in a new commit? That might be the only way to stop git from saying there is a change here.